assignment.md

```diff
@@ -11,47 +11,62 @@ If the version control concepts or `git` commands are still confusing to you, co
  0. Create a branch in this assignment repository named `submission`, then list all the branches you have (Hint: look at the Branch chapter of notes on how branches work, how to create new ones, and check existing branches):
 
     ```bash
-    # git command used and new list of branches
+    # git command used: `git checkout -b submissions
+    # list of branches: 
+	main
+	*submission
     ```
 
  1. List the current `remote` name and hyperlink destination of this assignment repository (Hint: if you are not sure, add `--help` after your command to check remotes.). 
 
     ```bash
-    # your git command and output here
+    # git command: git remote get-url origin
```

Comment:
This is very detailed, i like it. but `git remote -v` works too, just won't specifically only give you the url, but also the name of the remote.